mne/viz/epochs.py

```diff
@@ -143,88 +380,128 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         overlay_times = np.array(overlay_times)
         times_min = np.min(overlay_times)
         times_max = np.max(overlay_times)
-        if ((times_min < epochs.tmin) or (times_max > epochs.tmax)):
+        if ((times_min < epochs.times[0]) or (times_max > epochs.times[-1])):
```

Comment:
why is this necessary?

Reply:
You mean the line per se, or the change?

Reply:
Tbh I forgot why I changed it. I guess I had a problem with some dataset?